microapps/FeatureFormsApp/app/src/main/java/com/arcgismaps/toolkit/featureformsapp/screens/map/MapViewModel.kt

```diff
@@ -94,23 +96,22 @@ data class ErrorInfo(val fieldName: String, val error: FeatureFormValidationExce
 @HiltViewModel
 class MapViewModel @Inject constructor(
     savedStateHandle: SavedStateHandle,
-    private val portalItemRepository: PortalItemRepository
-) : ViewModel(),
-    MapInterface by MapInterfaceImpl(ArcGISMap()) {
+    portalItemRepository: PortalItemRepository,
+    val proxy: MapViewProxy,
+    private val application: Application,
+    @ApplicationScope private val scope: CoroutineScope
+) : ViewModel() {
```

Comment:
```suggestion
    application: Application,
    @ApplicationScope private val scope: CoroutineScope
) : AndroidViewModel(application) {
```
`AndroidViewModel` can be used here just to make it more explicit.

Reply:
I had to add a base class which takes only the singular application parameter to do this.